turborepo-remote-cache/templates/deployment.yaml

```diff
@@ -22,11 +22,22 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.pvc.create -}}
+      volumes:
+        - name: turborepo-remote-cache
+          hostPath:
```

Comment:
Is there any reason for mounting via hostPath? Won't it just mount a directory on a node and simulate network attached storage?

Reply:
I guess you'd rather want to refer to via pvc claim to actually use the provisioned pvc, a'la:
```
  volumes:
    - name: turborepo-remote-cache
      persistentVolumeClaim:
        claimName: turborepo-remote-cache
```

Reply:
Thanks for the review. Changed it now.